lib/Element/Base.php

```diff
@@ -73,7 +73,7 @@ public function xmlSerialize(Xml\Writer $writer): void
      * $reader->parseInnerTree() will parse the entire sub-tree, and advance to
      * the next element.
      *
-     * @return array<string, mixed>|string|null
+     * @return array<int,array<string, mixed>>|string|null
```

Comment:
This was an existing error in the PHPdoc. This thing returns an array of arrays.